wordpress-mobile/WordPress-iOS
# Issue/8213 site creation theme

Fixes #8213 

Needs Review: @nheagy

**To test:**
- Start Site Creation process.
- Select a Site Category.
- Verify the Themes displayed match those for the Site Category (listed here https://hogwartsp2.wordpress.com/2017/10/12/social-auth-signup-with-google-design-i3/#comment-6657)
- Verify the layout matches the design.

**Notes:**
- The step description is different than in the design, as discussed here https://hogwartsp2.wordpress.com/2017/11/27/social-auth-signup-with-google-design-i5/#comment-6951.
- The Themes for each category are hard-coded.
- For some reason, the Portfolio > Lodestar theme isn't being returned anymore in the fetch. Leaving that as-is as it should be addressed with the new endpoint.
- When a Theme is selected, this temporary alert appears:
<img width="309" alt="alert" src="https://user-images.githubusercontent.com/1816888/33403606-482994ae-d51e-11e7-861a-d8771cf9bfd6.png">

**Implemented:**

![implemented](https://user-images.githubusercontent.com/1816888/33403650-6ce5696c-d51e-11e7-99ed-434f0efba917.png)

**Designed:**

<img width="311" alt="design" src="https://user-images.githubusercontent.com/1816888/33403663-7b0fb768-d51e-11e7-9627-96b270de8a95.png">
